tools/clang/test/CodeGenSPIRV/type.line-stream.hlsl

```diff
@@ -0,0 +1,31 @@
+// Run: %dxc -T gs_6_0 -E main
+
+// CHECK: OpEntryPoint Geometry %main "main" %in_var_VertexID %gl_Position %out_var_COLOR0 %out_var_TEXCOORD0
```

Comment:
We can turn `in_var_VertexID` into `{{%\w+}}` here since we don't care about it here. Same for the rest.